database/migrations/2023_09_18_000000_rename_password_resets_table.php

```diff
@@ -0,0 +1,23 @@
+<?php
```

Comment:
Why is there a rename migration? Can we remove this and make sure the original migration is just correct?

Reply:
Part of Laravel Shift: https://github.com/spatie/Mailcoach/pull/63